.gitignore

```diff
@@ -13,3 +13,6 @@
 
 # Dependency directories
 vendor/
+
+# goland 
+.idea/*
```

Comment:
I'd like to keep the `.gitignore` sparse if we can -- @exdx maybe it's better for this to go in global git settings per contributor?